### ISSUE TITLE
Add ability to use timestamped transactional location data to identify potential parents for animals with a missing parent

### ISSUE THREAD

**Issue by rmsharp** [rmsharp/nprcgenekeepr#28]
Colocations at or near the birth date minus gestational duration can be identified with sufficient location data.

**Comment by rmsharp** [comment]
Tracked under umbrella #45 (*Principled parent identification via estimated conception date*) as the **deferred, larger** sub-task. It shares the umbrella's primitive (estimated conception date = birth − gestation) but, unlike #31, requires a **timestamped colocation/location data model that does not exist in the package today** — gated on the upstream data-pull issues #11 (Oracle) / #12 (ARMS) to define the source. Needs a written data-model spec (grain of 'colocation': same cage/room/enclosure; point-in-time vs. overlapping range) before sizing. Staying open as a distinct sub-task (not a duplicate of #31).
